src/components/columns/columns.component.tsx

```diff
@@ -36,11 +36,9 @@ export interface IFormioColumn {
   sizeMobile?: ColumnSize;
 }
 
-export interface IColumnComponent extends IFormioColumn {
-  defaultValue: undefined;
-
-  key: undefined;
+type IFormioColumnComponentSchema = IFormioColumn & ComponentSchema;
```

Comment:
I think this is more a naming thing and might need to be reconsidered. The goed here is to specify a renderable thing. Might be ComponentSchema, or a column. But it might be nicer her to narrow it only the specifics.